### DISCUSSION TITLE
单个 Doris Sink 同步多张表 是否提上日程了

### DISCUSSION THREAD

**Discussion by SamuelBin** [apache/doris#14283] [A - General / Q&A]
单个 Doris Sink 同步多张表 是否提上日程了

**Comment by dataalive** [comment]
这里具体点是指哪个工具吗？

**Reply by SamuelBin** [reply]
flink-doris- Connect 就是想使用一个doris sink能根据数据选择不同的表

**Reply by SamuelBin** [reply]
不过，我已经将doris sink 按自己的需求改了一下，能达到了一个sink，根据数据里面的某个字段去选择这条数据该存放那张表了

**Comment by hxy317** [comment]
> 不过，我已经将doris sink 按自己的需求改了一下，能达到了一个sink，根据数据里面的某个字段去选择这条数据该存放那张表了

求分享
